### PR TITLE
01_decouverte_micropython: small fixes and additions

### PR DESCRIPTION
**Pull request by Nhqml**
Fix some typos and add more explanations about conditions (especially
about the `elif` keyword).

### CODE REVIEW

**Comment by Nhqml** [comment]
Also I noticed some special characters (such as: è, é, à) are not rendered in the final pdf. However I could not understand why is that because *it works on my machine* :shrug:

**Comment by Nhqml** [comment]
Alright so I've investigated this problem, it seems that the version of `latexmk` (provided by `texlive-full` package) is kinda old (2015, WTF is that?). I believe this is what causes the special characters issue.

The only workaround I've found (since I don't think we're able to bump the version) is to write theses characters the *old way*, meaning that in order to get an `è` we would need to type ```\`e```. I'm working on doing this right now, a commit will follow soon.

File: 01_decouverte_micropython/01_decouverte_micropython.md [hunk 1]
```diff
@@ -345,11 +345,12 @@ calculer le score du joueur.
...
-lorsqu'une condition est vraie. Pour se faire la syntaxe est d'écrire `if
-condition:` suivi d'un bloc de code indenté. Le bloc de code en question ne sera
-alors exécuté que si `condition` s'évalue à `True`. Pour simplifier, un `if`
-peut être suivi d'un ou plusieurs `else:` qui exécute un bloc de code uniquement
-si la condition du `if` était fausse.
+lorsqu'une condition est vraie. Pour ce faire la syntaxe est : `if condition:`
+suivi d'un bloc de code indenté. Le bloc de code en question ne sera alors
+exécuté que si `condition` s'évalue à `True`. Un `if` peut être accompagné
+d'un `else` qui exécute un bloc de code uniquement si la condition du `if`
+était fausse. Enfin, il y a aussi le `elif` qui est la contraction du `else` et
```
Comment (Elfikurr) [inline 01_decouverte_micropython/01_decouverte_micropython.md:352]: s/était/est/
Je pense aussi qu'il faudrait rajouter une phrase un tout petit peu plus claire sur le elif, plus que "c'est un else et un if"

File: 01_decouverte_micropython/01_decouverte_micropython.md [hunk 2]
```diff
@@ -361,8 +362,10 @@ x = randint(0, 100)  # assigne un nombre aléatoire à x
 
 if x < 50:
     display.scroll('x est inférieur à 50')
+elif x < 80:
```
Comment (Elfikurr) [inline 01_decouverte_micropython/01_decouverte_micropython.md:365]: Peut-être rajouter un autre elif, pour montrer qu'on peut en avoir plusieurs ?
Ou alors en parler au dessus.

**Review by Elfikurr — CHANGES_REQUESTED** [review]
J'ai pas testé si ça s'affiche bien quand on le make, mais je pense que tu l'as fait. :)

Au pire on verra quand Travis le deploy.

Merci en tout cas !

**Review by Elfikurr — APPROVED** [review]

**Pull request merged**